starknet_py/utils/crypto/web3.py

```diff
@@ -0,0 +1,8 @@
+import re
+
+
+def is_valid_eth_address(address: str) -> bool:
```

Comment:
Let's make this a private function and put it under the `estimate_message_fee` method 👍